src/components/Modal/Modal.module.css

```diff
@@ -96,30 +94,39 @@
     }
 
     .content {
-      width: 100%;
+      /* min-width to prevent both this element (.content) and sibling element (.asideWrapper) to overflow their continers */
+      min-width: 0;
       height: 100%;
       display: flex;
       flex-direction: column;
+      padding: 0 var(--spacing-padding-sm, 8px) 0 var(--spacing-padding-xl, 24px);
```

Comment:
This could represent a problem.

In case the modal is opened without the aside panel, the only padding the modal will have is this one, causing the right padding to be only of 8px and the left padding to be of 24px. This could cause an effect where the content of the modal stretch to the right side far more than the right side.